File: Assignment35_5.c
// Write a program which accept string display it in reverse order
#include<stdio.h>

void Reverse(char *str)
{
    char *temp = str;

    while(*str != '\0')
    {
        str++;
    }

    str--;
    while(str >= temp)
    {
        printf("%c",*str);
        str--;
    }
    printf("\n");
}

int main()
{
    char Arr[] = {'\0'};
    int iRet = 0;

    printf("Enter a String: \n");
    scanf("%[^'\n']s",Arr);

    Reverse(Arr);

    return 0;
}

File: Assignment35_5.java
// Write a program which accept string display it in reverse order
import java.util.Scanner;
class Logic
{
    public void Reverse(String str)
    {
        char Arr[] = str.toCharArray();

        int iLength = Arr.length;
        int i = 0;
        for(i = iLength-1; i >= 0; i--)
        {
            System.out.print(Arr[i]);
        }
    }
}

class Assignment35_5
{
    public static void main(String[] args) 
    {
        Scanner scanobj = new Scanner(System.in);

        System.out.println("Enter a string: ");
        String sobj = scanobj.nextLine();

        Logic lobj = new Logic();
        lobj.Reverse(sobj);    

    }
}